xp/rel-jdk11-overrides.yaml

```diff
@@ -0,0 +1,17 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp10-openjdk11-openshift-rhel8"
+
+modules:
+      install:
+          - name: jboss.container.eap.galleon.build-settings
```

Comment:
Should you add the python cekit module like done in https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/EAP_730_SP01/dev-jdk11-overrides.yaml#L27 and https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/EAP_730_SP01/rel-jdk11-overrides.yaml#L10

Reply:
Yeah, thats probably a good idea, I'll double check that.

Reply:
Added